readme.txt

```diff
@@ -215,6 +215,12 @@ Still not happy? Shoot us an email to support@theeventscalendar.com or tweet to
 
 == Changelog ==
 
+= [4.6.24.1] 2018-10-08 =
```

Comment:
This places the 4.6.24.1 changelog above the 4.6.25 changelog entries ... we probably want it in the other order.